easybuild/easyconfigs/p/pftoolsV3/pftoolsV3-3.2.12-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,70 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+easyblock = 'CMakeMake'
+
+name = 'pftoolsV3'
+version = '3.2.12'
+
+homepage = 'https://github.com/sib-swiss/pftools3'
+description = """A suite of tools to build and search generalized profiles (protein and DNA)."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/sib-swiss/pftools3/archive/refs/tags/']
+checksums = ['1de33d6444015942afa690d54a103ad5a3d68f416fad63f247c12fa3f71a550c']  # pftools3-3.2.12.tar.gz
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.21.1'),
+    ('make', '4.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('libgd', '2.3.3'),
+    ('libharu', '2.3.0', '', ('foss', '2021b')),
```

Comment:
@Ghepardo please provide an easyconfig for this dependency with the correct toolchain, i.e. `GCCcore`, `11.2.0`, rather than forcing it this way.

Reply:
@verdurin I was under the impression the 'foss' toolchains were supersets of the 'GCCcore' ones, and were compatible.  Anyway, I've created a new easyconfig for the dependency libharu using GCCcore, and I've made this a dependency of pftoolsV3.  It looks as though these changes are going through your system automatically.